prow/github/client.go

```diff
@@ -1026,7 +1026,7 @@ func (c *Client) GetPullRequests(org, repo string) ([]PullRequest, error) {
 	path := fmt.Sprintf("/repos/%s/%s/pulls", org, repo)
 	err := c.readPaginatedResults(
 		path,
-		"application/vnd.github.symmetra-preview+json", // allow the description field -- https://developer.github.com/changes/2018-02-22-label-description-search-preview/
+		"application/vnd.github.symmetra-preview+json, application/vnd.github.shadow-cat-preview", // allow the description field -- https://developer.github.com/changes/2018-02-22-label-description-search-preview/
```

Comment:
We don't need these, right?

Reply:
Not explicitly for this plugin, but I figured that we should be asking for the same fields as we are returned by a webhook, whenever possible. If we don't ask for these, but are defining them in our types, then it will just default to false in any other operation.

Reply:
I think we do: https://developer.github.com/v3/pulls/#list-pull-requests

nit: Please update the comment to indicate that the new value enables the `draft` field.

Reply:
@cjwagner we get the field regardless in a webhook delivery